os/arch/arm/src/amebasmart/amebasmart_i2s.c

```diff
@@ -200,8 +198,32 @@ struct amebasmart_i2s_s {
 	struct amebasmart_buffer_s *freelist_tx; /* A list a free buffer containers */
 	struct amebasmart_buffer_s containers_tx[I2S_DMA_PAGE_NUM];
 #endif
+};
 
+/* I2S device structures */
+static const struct amebasmart_i2s_config_s amebasmart_i2s2_config = {
+	.i2s_mclk_pin = PB_22,
+	.i2s_bclk_pin = PB_21,
+	.i2s_ws_pin = PA_16,
+	.i2s_sd_tx_pin = PB_10,
+	.i2s_sd_rx_pin = PB_19,
+
+	.i2s_idx = I2S_NUM_2;
+	.rxenab = 0;
+	.txenab = 1;
+};
 
+static const struct amebasmart_i2s_config_s amebasmart_i2s3_config = {
+	.i2s_mclk_pin = PA_15,
+	.i2s_bclk_pin = PA_14,
+	.i2s_ws_pin = PA_13,
+	/* Question To Samsung, which pins to use, althoug it's an RX only I2S */
+	.i2s_sd_tx_pin = PB_11,
+	.i2s_sd_rx_pin = PB_20,
+
+	.i2s_idx = I2S_NUM_3;
+	.rxenab = 1;
+	.txenab = 0;
 };
```

Comment:
Many lines of code below are using priv->txenab direclty. Please update it use priv->config->txenab and so on